vochain/scrutinizer/indexertypes/types.go

```diff
@@ -325,6 +325,14 @@ func (b *BlockMetadata) String() string {
 	return builder.String()
 }
 
+type TokenTransfer struct {
+	ID        int64          `json:"id"`
+	From      types.HexBytes `json:"from,omitempty"`
+	To        types.HexBytes `json:"to"`
+	Amount    uint64         `json:"amount"`
+	Timestamp time.Time      `json:"timestamp"`
```

Comment:
I think we might add "height", its cheap and can be useful for finding the transaction.

Reply:
yup, good one, so we can have the block number